core/src/main/resources/schema/infinispan-config-12.0.xsd

```diff
@@ -2055,6 +2055,14 @@
         </xs:annotation>
       </xs:element>
     </xs:sequence>
+    <xs:attribute name="conflict-resolver" type="xs:string">
+      <xs:annotation>
+        <xs:documentation>
+          Specifies the fully qualified name of a class that implements the
+          XSiteConflictResolver interface. Use if for ASYNC strategy backup.
```

Comment:
The xsd needs updating to specify the available enum values, as well as the ability to add a custom implementation.